doc/api/events.markdown

```diff
@@ -201,7 +201,7 @@ added and `'removeListener'` when a listener is removed.
 
 ### Event: 'newListener'
 
-* `event` {String|Symbol} The event name
+* `eventName` {String|Symbol} The name of the event listened to
```

Comment:
`The name of the event listened to` doesn't sound correct. Maybe `The name of the event being listened for`.